.github/workflows/client-android.yml

```diff
@@ -152,6 +152,7 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: AKIAJ3SWUQ4QLNQC7FXA
           AWS_SECRET_ACCESS_KEY: ${{ secrets.android_client_build_aws_secret_key }}
+          AWS_DEFAULT_REGION: 'us-east-1'
```

Comment:
Hmm, maybe it could be better to add this env in the scripts instead of the workflows? 🤔 Usually the bucket is specified there, so maybe it would be more appropriate, what do you think?

Reply:
I'm not sure about that, this would require to split the AWS variables into two places, which might be confusing a bit. 

Maybe we can think of using the dedicated action instead of setting those envs manually:
* https://github.com/marketplace/actions/configure-aws-credentials-action-for-github-actions

Reply:
Ok, let's leave it as is for now